### ISSUE TITLE
ADA Compliance - Video Controls Not Accessible 

### ISSUE THREAD

**Issue by verilucis** [appleple/modal-video#92]
Hi there! 

First, thanks for the work on modal-video.js, it's appreciated. 

Working on making this ADA compliant.  When the modal opens, the focus trap and one other item appear to tabbable.  Am I missing anything here or is it possible to make the video and it's controls tabbable, like a regular embedded on page youtube video, and still keep the focus trap?   

Thanks again, and hope you are you doing well. :)
